example_docs/docs/sphinx_guide/custom_directives.rst

```diff
@@ -0,0 +1,51 @@
+=================
+Custom Directives
+=================
+
+customcalloutitem
+=================
+
+.. raw:: html
+
+   <div class="tutorials-callout-container">
+      <div class="row">
+
+.. customcalloutitem::
+   :header: Go back to qiskit.org!
+   :description: That's a cool site too :)
+   :button_link:  https://qiskit.org
+   :button_text: qiskit.org
+
+.. raw:: html
+
+      </div>
+   </div>
+
+
+customcarditem
+==============
+
+.. raw:: html
```

Comment:
Ditto that this is not included in the `customcarditem` because it's common to have multiple "items" in a group. This is the outer container code.